doc/articles/features/using-skia-gtk.md

```diff
@@ -23,4 +23,18 @@ It may be required, depending on the environment, to use software rendering.
 To do so, immediately before the line `host.Run()` in you `main.cs` file, add the following:
 ```
 host.RenderSurfaceType = RenderSurfaceType.Software;
-```
\ No newline at end of file
+```
+
+## Upgrading to a later version of SkiaSharp
+
+By default Uno comes with a set of **SkiaSharp** dependencies set by the **[Uno.UI.Runtime.Skia.Gtk](https://nuget.info/packages/Uno.UI.Runtime.Skia.Gtk)** package. 
+
+If you want to upgrade **SkiaSharp** to a later version, you'll need to specificy all packages individually in your project as follows:
+
+```xml
+<ItemGroup>
+   <PackagReference Include="SkiaSharp" Version="2.88.0" /> 
+   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.0" /> 
+   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0" /> 
```

Comment:
Good point. It will also be needed in the gtk head as well.
https://github.com/unoplatform/uno/blob/025a5af1d101050ee4085c1a7e12a753c7b243ed/src/Uno.UI.Runtime.Skia.Gtk/Uno.UI.Runtime.Skia.Gtk.csproj#L38-L43